code/script.js

```diff
@@ -1,53 +1,216 @@
+
+let selectedDish;
+let portion;
+let userName;
+let finalOrder;
+
+let pop = document.querySelector('#audio')
+pop.volume = 0.5;
+
 
 // Functions goes here 👇
 
-// A function that will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
-  // The if statement checks if the sender is the user and if that's the case it inserts
-  // an HTML section inside the chat with the posted message from the user
-  if (sender === 'user') {
+  if (sender === "user") {
+    pop.play();
     chat.innerHTML += `
-      <section class="user-msg">
-        <div class="bubble user-bubble">
-          <p>${message}</p>
-        </div>
-        <img src="assets/user.png" alt="User" />  
-      </section>
-    `
-    // The else if statement checks if the sender is the bot and if that's the case it inserts
-    // an HTML section inside the chat with the posted message from the bot
-  } else if (sender === 'bot') {
+        <section class="user-msg">
+          <div class="bubble user-bubble">
+            <p>${message}</p>
+          </div>
+          <img src="assets/user.png" alt="User" />  
+        </section>
+      `;
+  } else if (sender === "bot") {
+    pop.play();
     chat.innerHTML += `
-      <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
-        </div>
-      </section>
-    `
+        <section class="bot-msg">
+          <img src="assets/bot.png" alt="Bot" />
+          <div class="bubble bot-bubble">
+            <p>${message}</p>
+          </div>
+        </section>
+      `;
   }
+  chat.scrollTop = chat.scrollHeight;
+};
+
+const greetUser = () => {
+  nameInput.style.disply = "flex";
+  showMessage("Hello there, what's your name?", "bot");
+};
+
+
+const handleNameInput = (event) => {
+  event.preventDefault();
+  userName = nameInput.value;
+  showMessage(userName, "user");
+  nameInput.value = "";
+  chooseFoodOption(userName);
+};
+
+const chooseFoodOption = (userName) => {
+  foodCategory.style.display = "flex";
+  nameWrapper.style.display = "none";
+  showMessage(`Hi ${userName}! What would you like to order? `, "bot");
+};
+
+const chooseSubOption = (category) => {
+  foodCategory.style.display = "none";
+
+  if (category == "pizza") {
+    subPizza.style.display = "flex";
+    showMessage("Pizza", "user");
+  } else if (category == "salad") {
+    subSalad.style.display = "flex";
+    showMessage("Salad", "user");
+  } else {
+    subKebab.style.display = "flex";
+    showMessage("Kebab", "user");
+  }
+  showMessage(
+    `You chose ${category}. What kind of ${category} would you like?`,
+    "bot"
+  );
+};
+
+const finalFoodChoice = (chooseSubOption) => {
+  selectedDish = chooseSubOption;
+  showMessage(chooseSubOption, "user");
+  showMessage(`You chose ${chooseSubOption}. Large or X-large portion?`, "bot");
+  portionSelect();
+};
+
+const portionSelect = () => {
```

Comment:
To follow the naming pattern the other functions are following, maybe this one could be called selectSize?